README.md

```diff
@@ -434,7 +434,10 @@ On MacOS
 - AppleClang 11.0
 
 On Windows
-- Visual Studio 2017
+- Visual Studio 2019
```

Comment:
This change does not change what is tested, so it should not change the list of compilers.